repository/good_repo_entity.py

```diff
@@ -0,0 +1,19 @@
+import sqlalchemy as sa
+from sqlalchemy.orm import declarative_base
+from sqlalchemy import Integer, String
+
+Base = declarative_base()
+
+class GoodItem(Base):
```

Comment:
```suggestion
class GoodItemModel(Base):
```

часто для orm model используют постфикс model